packages/types/src/events/assistant.ts

```diff
@@ -0,0 +1,29 @@
+export interface AssistantThreadStartedEvent {
+  type: "assistant_thread_started";
+  assistant_thread: {
+      user_id: string;
+      context: { 
+        channel_id: string; 
+        team_id: string; 
+        enterprise_id: string;
```

Comment:
the `context` property can be empty when an end-user started a convo next to non-channel views such as "Unread", "Threads", and others
```suggestion
        channel_id?: string; 
        team_id?: string; 
        enterprise_id?: string;
```